src/server/req_manager.c

```diff
@@ -1561,17 +1612,27 @@ mgr_sched_set(struct batch_request *preq)
 	int	  bad_attr = 0;
 	svrattrl *plist;
 	int	  rc;
+	pbs_sched *psched;
+
+	psched = find_scheduler(preq->rq_ind.rq_manager.rq_objname);
+	if (!psched) {
+		req_reject(PBSE_UNKSCHED, 0, preq);
+		return;
+	}
+	if (validate_sched_attr_vals(preq, psched))
```

Comment:
Well, validate_sched_attr_vals validates sched_priv and sched_log directory. It iterate through list of scheduler objects. Some thing unusual than other validation. 

Since @arungrover and @bhroam of the same opinion, I may be wrong here. Will move these changes to action function. 

BTW, in request manager for other objects like server, node etc validation is not done in action function.